src/structures/breakend.cpp

```diff
@@ -0,0 +1,28 @@
+#include "structures/breakend.hpp"
+
+#include <cstdint>  // for enum struct _ : uint8_t
```

Comment:
std includes above internal ones

Reply:
And this can be probably omitted, at least I have never included anything for `uint8_t`... Maybe try?